src/Sulu/Bundle/AdminBundle/Resources/public/js/components/csv-export/main.js

```diff
@@ -12,116 +12,15 @@
  * @class CSV-Export
  * @constructor
  */
-define(['jquery', 'underscore', 'text!./form.html'], function($, _, form) {
+define(['text!./form.html'], function(form) {
```

Comment:
Hm, good point... makes everything a bit slower because there is another request, but I think that's worth it. I'll implement it.